### ISSUE TITLE
`--profile=gpu-hq` with `--gpu-context=wayland` leads to awful performance

### ISSUE THREAD

**Issue by Gigas002** [mpv-player/mpv#11753]
### Important Information

mpv version:

```
mpv 0.35.0-419-gf79458476b Copyright © 2000-2023 mpv/MPlayer/mplayer2 projects
 built on Wed May 31 09:33:42 2023
libplacebo version: v5.264.0-277-ge6846192
FFmpeg version: n6.0
FFmpeg library versions:
   libavutil       58.2.100
   libavcodec      60.3.100
   libavformat     60.3.100
   libswscale      7.1.100
   libavfilter     9.3.100
   libswresample   4.10.100

```

linux destribution and version:

```
Operating System: Manjaro Linux
Kernel: Linux 6.3.5-x64v3-xanmod1
Architecture: x86-64
```

source of mpv binary: mpv-git from AUR: https://aur.archlinux.org/packages/mpv-git/

DM: wayland, KDE Plasma 5.27.5

dGPU: NVIDIA GeForce RTX 3060 Laptop; proprietary nvidia driver 530.41.03

iGPU: amdgpu, mesa 23.1.1

### Reproduction steps

```
mpv --no-config --gpu-context=wayland --profile=gpu-hq --log-file=output.txt vid.mp4 
```

### Expected behavior

Video plays without drops, as on `x11` session

### Actual behavior

The whole system becomes very laggy

### Log file

https://0x0.st/HbLA.txt

### Additional details

Also noticed, that `deband`, and especially `interpolation` has bad performance as well. [My config](https://github.com/Gigas002/software-and-services/blob/master/mpv/mpv-x11.conf) runs flawlessly on x11 session, but all the above mentioned stuff must be commented out to have the same performance on wayland.

**Comment by Dudemanguy** [comment]
According to this log, you're using the iGPU here. When you run it on X11, do you normally use the nvidia card or the amd one?

**Comment by Gigas002** [comment]
Fair point. I use `x11vk` on `x11` session, so it automatically grabs my dGPU. I've forced mpv to run on nvidia with `prime-run` and it is in fact works pretty good. Sorry I didn't check this out before posting...
Regarding the second way to hit the performance, I found out, that it's a `video-sync=display-resample` fault. It's not that bad, but still the dropping frames are a bit noticable:

https://0x0.st/Hb91.txt
